api/peer_connection_interface.h

```diff
@@ -91,7 +91,7 @@
 #include "api/data_channel_interface.h"
 #include "api/dtls_transport_interface.h"
 #include "api/fec_controller.h"
-#include "api/ice_gatherer_interface.h"
```

Comment:
It is compiling without this, but this was part of the ICE Forking addition. Maybe we should keep it?